blockchain/stake/staketx.go

```diff
@@ -1193,3 +1193,13 @@ func SetTxTree(tx *dcrutil.Tx) {
 
 	tx.SetTree(indicatedTree)
 }
+
+// IsStakeSubmissionTxOut indicates whether the txOut identified by the given
```

Comment:
please word wrap this comment so it flows as a single paragraph (see the rest of the codebase as an example)